### PR TITLE
153 [Webinars] Webinar detail page with Speakers Block

### LINKED ISSUE #153: [Webinars] Webinar detail page with Speakers block
image

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #153 

Test URLs:
- Before: https://main--danaher-abcam--aemsites.hlx.live/en-us
- After: https://153-webinar-speakers-block--danaher-abcam--aemsites.hlx.page/en-us/webinars/test-page

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
Approved how it looks on the front end, if @rgravitvl is happy with the JS chagne.

**Review by pardeepgera23 — APPROVED** [review]

File: eds/blocks/speakers-info/speakers-info.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  createOptimizedPicture, decorateIcons,
+} from '../../scripts/aem.js';
+import {
+  div, span,
+} from '../../scripts/dom-builder.js';
+
+export default function decorate(block) {
+  const parentBlock = block.parentElement.parentElement.firstElementChild;
+  const parentBlockTitle = parentBlock.querySelector('p');
+  parentBlockTitle.classList.add(...'text-black text-2xl font-medium leading-7'.split(' '));
```
Comment (rgravitvl) [inline eds/blocks/speakers-info/speakers-info.js:11]: Please use applyClasses from script JS

File: eds/scripts/scripts.js [hunk 2]
```diff
@@ -311,6 +312,21 @@ function decorateStoryPage(main) {
   }
 }
 
+function decorateWebinarPage(main){
```
Comment (rgravitvl) [inline eds/scripts/scripts.js:315]: Can we do this in template JS??

File: eds/templates/webinars/webinars.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  buildBlock, getMetadata,
+} from '../../scripts/aem.js';
+import {
+  div,
+} from '../../scripts/dom-builder.js';
+
+export default async function buildAutoBlocks() {
+  const main = document.querySelector('main');
+  const sectionColumns = main.querySelector(':scope > div > div.columns')?.parentElement;
+  const headingDiv = div({ class: 'webinar leading-6 text-2xl font-bold text-black px-6 lg:px-80 mb-0' }, 'Abcam webinar');
```
Comment (rgravitvl) [inline eds/templates/webinars/webinars.js:11]: heading should be added from UE

**Review by rgravitvl — DISMISSED** [review]

File: eds/templates/webinars/webinars.js (hunk 3 above)
Comment (sharanagoudapatil) [reply eds/templates/webinars/webinars.js]: FYI - Heading was removed from the Figma itself. I've removed this extra code from webinars.js

**Review by rgravitvl — APPROVED** [review]

**Pull request merged**